enterprise/admin/views.py

```diff
@@ -134,15 +134,14 @@ def _build_context(self, request, customer_uuid):
         search_keyword = self.get_search_keyword(request)
         linked_learners = self.get_enterprise_customer_user_queryset(search_keyword, customer_uuid)
         pending_linked_learners = self.get_pending_users_queryset(search_keyword, customer_uuid)
+        enrollment_url = '/api/enrollment/v1/'
```

Comment:
Add `LMS_ENROLLMENT_API_PATH` to settings.

Reply:
ack

Reply:
Perhaps it is better to leave this here after all.